network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html

```diff
@@ -249,7 +249,7 @@ <h2>{% trans 'Three months later' %}</h2>
 
 {# Create curved effect on section using border radius #}
 <div class="tw-rounded-b-[30px] tw-mb-[-30px]">
-    <section class="container tw-max-w-4xl tw-text-center tw-my-7 large:tw-my-[80px] large:tw-my-[120px]" data-scroll-section>
+    <section class="container tw-max-w-4xl tw-text-center tw-my-24 large:tw-my-[120px]" data-scroll-section>
```

Comment:
Reminder for @mmmavis : check if we should be keeping `large:tw-my-[80px]` or `large:tw-my-[120px]`

Reply:
I did a quick test by adding `large:tw-my-[80px] large:tw-my-[120px]` to a `<div>`. It ends up having top/bottom margin of 80px. In this case I think we should be keeping `large:tw-my-[80px]` instead.

```suggestion
    <section class="container tw-max-w-4xl tw-text-center tw-my-24 large:tw-my-40" data-scroll-section>
```